app/views/editor/rich.html.erb

```diff
@@ -404,6 +406,29 @@
       }
     });
 
+    editor.mapModule.blurredLocation.setZoomByPrecision(getPrecisionFromNum("<%= @lat %>"));
```

Comment:
Awesome @nstjean ! 
But this `getPrecisionFromNum` function needs to be removed. I have added more details here: https://github.com/publiclab/plots2/issues/6946

Briefly - the zoom level calculation needs to be inside a function in LBL.
The PL.editor will just take that zoom level as input and just set map to that zoom in map_module of PL.editor.
If zoom is present in the form of zoom:xx in plots2, let's pass that otherwise call the function of LBL to get zoom and then pass it. Thanks 😄 !!!

Reply:
Ahhh, ok! That makes sense!